src/modules/missionHelper/i18n/en_GB.root.json

```diff
@@ -44,6 +44,10 @@
 			"description": "Hides Battalion Chief completely and shows only MCV.",
 			"title": "Hide Battalion Chiefs"
 		},
+		"hoverTip": {
+				"description": "Tooltips for as example the overlay button.",
```

Comment:
Again, are the indentations right?

Reply:
they are then set correctly during the build :)